gui-wm/dwl/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person" proxied="yes">
+        <email>leohdz172@proton.me</email>
+        <name>Leonardo Hernández Hernández</name>
+        <description>Upstream Maintainer (please CC on bugs)</description>
```

Comment:
you're already going to be assigned them, so just drop this bit

or change it to "Additionally the upstream maintainer"